src/js/WebParts/ProjectStats/ChartConfiguration.ts

```diff
@@ -28,7 +28,7 @@ export default class ChartConfiguration {
     public showPercentage: boolean;
     public showItemSelector: boolean;
     private _pnpList: List;
-    private _contentTypes: any[];
+    // private _contentTypes: any[];
```

Comment:
Vi vil ikke ha utkommentert kode. Fjern det eller stå for det.

Reply:
Utkommentert kode er no-go.

Reply:
Disse har jeg rett og slett glemt å fjerne. Har tatt de vekk nå i commit [ba8cb3c](https://github.com/Puzzlepart/prosjektportalen/pull/658/commits/ba8cb3cedc31e775b4d3a9f52be6518f0db43d34)